samples/nfc/system_off/src/main.c

```diff
@@ -138,12 +138,26 @@ static void system_off(struct k_work *work)
 
 	dk_set_led_off(SYSTEM_ON_LED);
 
-	int rc = pm_device_action_run(cons, PM_DEVICE_ACTION_SUSPEND);
+	if (IS_ENABLED(CONFIG_PM_DEVICE) && IS_ENABLED(CONFIG_SERIAL)) {
+		static const struct device *dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+		int err;
+		enum pm_device_state state;
 
-	if (rc < 0) {
-		printk("Could not suspend console (%d)\n", rc);
+		if (dev) {
```

Comment:
Does the system go to the suspended state automatically?

Reply:
The runtime PM will disable the UART after the last byte has been sent.